kolibri/tasks/api.py

```diff
@@ -26,6 +27,13 @@
 
 logging = logger.getLogger(__name__)
 
+NETWORK_ERROR_STRING = _("There was a network error.")
```

Comment:
Slightly misleading variable names, as these are not strings, but gettext_lazy objects?

Reply:
These variables are temporary, as I'm looking to use them in places we actually raise the errors on.